ios/RNEngageBySailthru.m

```diff
@@ -0,0 +1,260 @@
+
+#import "RNEngageBySailthru.h"
+#import <UserNotifications/UserNotifications.h>
+
+@interface MARPurchase ()
+
+- (nullable instancetype)initWithDictionary:(NSDictionary *)dictionary;
+
+@end
+
+@implementation RNEngageBySailthru
+
+RCT_EXPORT_MODULE();
+
+- (EngageBySailthru *)engageBySailthruWithResolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject {
```

Comment:
Since this never actually uses the resolver, we can remove it to simplify this method with only the reject block.